app/controllers/houses_controller.rb

```diff
@@ -1,11 +1,11 @@
-class HouseController < ApplicationController
+class HousesController < ApplicationController
 
   def index
-    @houses = Houses.all
+    @houses = House.all
   end
 
   def show
-    @houses = Houses.all
+    @houses = House.find(params[:id])
```

Comment:
Would make more sense to call this variable `@house` since it will contain a single instance of a house